handlers/handler.go

```diff
@@ -2,6 +2,14 @@ package handlers
 
 import (
 	"net/http"
+	"time"
```

Comment:
Please sort the includes according to ( in all other files too ):

1. stdlib
empty line
2. outside libraries
empty line
3. external gaia libraries
empty line
4. internal gaia libraries.

Reply:
Just need to sort my fmt out, np :)

Reply:
@Skarlso are you doing these manually or using a formatter?

Reply:
By hand, I'm afraid. 😁